src/Chapter1.hs

```diff
@@ -541,8 +538,8 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+import Data.List (sort)
```

Comment:
Imports in Haskell should go after the `module ... where` and before the first definition of a function. Unfortunately, you are not allowed to write `import` statements between function definitions. For this to work, you need to move this import up. We explain `imports` in more details in the next chapter.